### PR TITLE
Create API endpoint for transaction rejection (DELETE) - Closes #167 #168

### LINKED ISSUE #167: Request multisignature transaction rejection
### Description

Create a microservice method `multisig.reject` endpoint for transaction rejection. 

### Motivation

- This method is needed to reject the transaction by one of participants.

### Acceptance Criteria

- The database gets updated with `rejected` flag
- Functional tests are written

### LINKED ISSUE #168: Create API endpoint for transaction rejection (DELETE)
### Description

Create an API endpoint for transaction update. 

In order to update an initiated transaction, a `DELETE` request to `/api/v2/transaction/multisig` has to be sent.

This endpoint has to call `multisig.reject` action.

At this point there is no validation performed, the microservice accepts all payloads within the defined schema.

### Motivation

A participant should be able to reject a transaction.

### Acceptance Criteria

- It is possible to find a transaction by serviceId and update the corresponding `rejected` flag
- Integration tests are ready

#### Request

`DELETE /api/v2/transactions/multisig`
`delete.transactions.multisig`

##### Schema

In order to reject an initiated transaction, a DELETE request with the following body has to be sent.

```jsonc
{
  "serviceId": <String>,
  "signatures": [{
    "publicKey": <String>,
    "signature": <String>,
  }]
}
```

#### Response

`200 OK`

```jsonc
{
  "data": [{
      "serviceId": <String>, // LS internal transaction ID (UUID)
      "nonce": <String>,
      "senderPublicKey": <String>,
      "asset": { ... },
      "moduleAssetId": <Number>,
      "fee": <String>,
      "expires": <Number>, // UNIX Timestamp
      "signatures": [
        {
          "publicKey": <String>,
          "address": <String>,
          "signature": <String>,
          "accountRole": <String> // mandatory/optional/owner
        }
      ]
  }],
  "meta": {
      "count": <Number>, "offset": <Number>, "total": <Number>,
  }
}
```

### PR DESCRIPTION
**Pull request by nagdahimanshu**
### What was the problem?
This PR resolves #167 #168

### How was it solved?
- [x] Add gateway method for multisig transaction delete
- [x] Add integration tests
- [x] Add/Fix swagger
- [x] Add `rejectMultisignatureTx` method to controller
- [x] Add `multisig.reject` action
- [x] Add implementation to reject multisig transaction, update `rejected` flag by `serviceId` in DB
- [x] Add Functional test for `multisig.reject` action

### How was it tested?
Local + Jenkins

### CODE REVIEW

**Review by MichalTuleja — CHANGES_REQUESTED** [review]
Overall looks good. Minor remarks below.

- Please pay attention to filenames in the gateway api methods i.e `deleteMultisigTransactions.js` -> `transactionMultisigReject.js`.

- Please update functional tests.

File: tests/integration/api_v2/http/multisigTransactions.test.js [hunk 1]
```diff
@@ -110,5 +110,19 @@ describe('Multisignature Transactions API', () => {
...
+			it('Reject a multisignature transaction', async () => {
+				const response = await api.del(`${endpoint}`, {
+					serviceId: refTransaction.serviceId,
+					signatures: inputTransaction.signatures,
+				});
+				expect(response).toMap(goodRequestSchema);
+				expect(response.data).toBeInstanceOf(Array);
+				expect(response.data.length).toBe(1);
+				response.data.forEach(multisigTxn => expect(multisigTxn).toMap(multisigTransactionSchema));
+				expect(response.meta).toMap(metaSchema);
```
Comment (sameersubudhi) [inline tests/integration/api_v2/http/multisigTransactions.test.js:124]: We do not check for the `rejected` flag within the return response. @MichalTuleja don't we need to map it in the response object?

File: tests/integration/api_v2/rpc/multisigTransactions.test.js [hunk 2]
```diff
@@ -120,5 +121,21 @@ describe('Multisignature Transactions API', () => {
...
+					signatures: inputTransaction.signatures,
+				});
+				expect(response).toMap(jsonRpcEnvelopeSchema);
+				const { result } = response;
+				expect(result.data).toBeInstanceOf(Array);
+				expect(result.data.length).toBe(1);
+				expect(response.result).toMap(resultEnvelopeSchema);
+				result.data.forEach(multisigTxn => expect(multisigTxn).toMap(multisigTransactionSchema));
+				expect(result.meta).toMap(metaSchema);
+			});
```
Comment (sameersubudhi) [inline tests/integration/api_v2/rpc/multisigTransactions.test.js:138]: Same as above.

**Review by sameersubudhi — APPROVED** [review]
LGTM 👍

File: tests/integration/api_v2/http/multisigTransactions.test.js (hunk 1 above)
Comment (MichalTuleja) [reply tests/integration/api_v2/http/multisigTransactions.test.js:124]: I think there should be a check against it.

**Review by MichalTuleja — CHANGES_REQUESTED** [review]
It looks we are almost there, but the functional tests remain unchanged. Please see line 65 in `multisig.action.test.js`, there could be a condition for the property `rejected`.

File: tests/integration/api_v2/http/multisigTransactions.test.js (hunk 1 above)
Comment (nagdahimanshu) [reply tests/integration/api_v2/http/multisigTransactions.test.js:127]: I mapped the `rejected` flag in the response and updated the corresponding integration test case to verify.

**Review by MichalTuleja — APPROVED** [review]

**Pull request merged**